test/CMakeLists.txt

```diff
@@ -40,6 +40,13 @@ macro(onedpl_add_test test_source_file)
         target_compile_options(${_test_name} PRIVATE $<$<CONFIG:Release>:-fno-strict-aliasing>)
     endif()
 
+    # oneDPL tests of std::complex required generates code to do JIT compilation of a kernel only when it is called
+    if ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang|IntelLLVM" )
```

Comment:
1) In case of "pure" clang what we have got in result?
2) The same changes (setting the option)  should be added into make files as well - oneDPL\make directory.
3) Please make sure that the changes don't break a test launch with not dcpp backend and other compilers (gcc and others).

Reply:
For cmake command
```
cmake -DCMAKE_BUILD_TYPE=debug -DONEDPL_BACKEND=dpcpp -DCMAKE_CXX_COMPILER=gcc -DONEDPL_DEVICE_TYPE=GPU ..
```
we receive message about fsycl option not supported.

For cmake command
```
cmake -DCMAKE_BUILD_TYPE=debug -DONEDPL_BACKEND=tbb -DCMAKE_CXX_COMPILER=gcc -DONEDPL_DEVICE_TYPE=GPU ..
```
cmake run without errors but skip some tests as required dpcpp backend.